libco/libco.c

```diff
@@ -29,8 +29,8 @@ void genode_free_secondary_stack(void *stack);
     #include "ppc.c"
   #elif defined(__aarch64__)
     #include "aarch64.c"
-  #elif defined(PSP)
-    #include "psp1.c"
```

Comment:
Maybe delete this file if it doesn't exist anymore?

Reply:
Done